python/ccxt/base/exchange.py

```diff
@@ -2638,3 +2638,18 @@ def handle_market_type_and_params(self, method_name, market=None, params={}):
         type = self.safe_string_2(params, 'defaultType', 'type', market_type)
         params = self.omit(params, ['defaultType', 'type'])
         return [type, params]
+
+    def fetchTradingLimits(self, symbols=None, params={}):
```

Comment:
This has to be overrided for coroutines in the async version in ccxt/async_support/base/exchange.py.

Reply:
how do we test the async methods?

Reply:
1. If the unified method is a part of the existing testset:
```bash
node js/test/test exchange_id
python3 python/ccxt/test/test_async.py exchange_id
php -f php/test/test_async.php exchange_id
```
2. If we don't have a corresponding test yet – we should make it, or we should make an ad-hoc script for the testing purpose, in-place.